src/config.ts

```diff
@@ -44,7 +44,7 @@ export const fhirConfig: FhirConfig = {
             service: 'OAuth',
             oauth: {
                 authorizationUrl: OAuthUrl,

```

Comment:
```suggestion
                authorizationUrl:`${OAuthUrl}/oauth2/authorize`,

```

Need to do the other url too: https://docs.aws.amazon.com/cognito/latest/developerguide/authorization-endpoint.html

Reply:
updated